sdk/daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/ledger/EventId.scala

```diff
@@ -7,41 +7,17 @@ package ledger
 import com.digitalasset.daml.lf.data.Ref._
 import com.digitalasset.daml.lf.transaction.NodeId
 
-import scala.util.Try
-
 // transactionId should be small so the concatenation in toLedgerString does not exceed 255 chars
```

Comment:
```suggestion
// transactionOffset should be small so the concatenation in toLedgerString does not exceed 255 chars
```

As we're using ledger offsets now in place of transaction IDs, does this comment still look reasonable?

Reply:
Actually, as we're now using longs, this comment may even now be trivially true?

Reply:
Ideally we would like to get rid of event id as a notion. However, scenario ledger seems to be based on it e.g. event id is passed as eventid in https://github.com/digital-asset/daml/blob/4c96c1deeeb234801f684bec5646a2a401991be3/sdk/compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Conversions.scala#L499

Reply:
@atriantafyllos-da I was looking at this comment/goal and the slack thread where @remyhaemmerle-da commented. I was wondering if we should simply move `EventId` into `ScenarioLedger` (which I interpret as a variant of the "make it private" PoV he expressed)?

Doing this might make it clear that scenario service is only using this and simplify the change set you might ow have on your hands.

@samuel-williams-da does this make sense to yourself?

Reply:
I'd be okay with this, I do wonder if we can drop it for something else entirely, but we can always do that later, I dont have time right now to think of something cleaner